ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteExporter.java

```diff
@@ -475,10 +475,17 @@ private static void convertTieLine(UcteNetwork ucteNetwork, TieLine tieLine, Uct
         // FIXME(mathbagu) REAL or EQUIVALENT ?
         UcteElementStatus status = tieLine.getTerminal1().isConnected() && tieLine.getTerminal2().isConnected() ? EQUIVALENT_ELEMENT_IN_OPERATION : EQUIVALENT_ELEMENT_OUT_OF_OPERATION;
 
+        String elementName = tieLine.getProperty(ELEMENT_NAME_PROPERTY_KEY, null);
+        String elementName1 = elementName;
+        String elementName2 = elementName;
+        if (elementName != null && elementName.contains(",")) {
+            elementName1 = elementName.split(",")[0];
+            elementName2 = elementName.split(",")[1];
+        }
```

Comment:
The property separator should be a constant, just in case this is not well chosen and we have to change it.